extensions/aproc/proc/ingest/drivers/impl/image_driver_helper.py

```diff
@@ -136,12 +144,13 @@ def to_item(driver: IngestDriver, itemFormat: ItemFormat, assetFormat: AssetForm
         item.properties.constellation = None
         item.properties.sensor = None
         image_asset: Asset = item.assets.get(Role.data.value)
-        if os.path.exists(url):
-            image_asset.size = os.stat(url).st_size
-        else:
-            driver.LOGGER.warn("{} does not exist, size not found".format(url))
+        image_asset.size = os.stat(local_url).st_size
         image_asset.airs__managed = False
-        image_asset.asset_format = assetFormat.value
+
+        image_asset.asset_format = asset_format.value
         image_asset.asset_type = ResourceType.gridded.value
         item.properties.eo__bands = bands
+
+        if local_url != url:
+            os.remove(local_url)
```

Comment:
The system should never delete any file. If it is really needed, then this must be documented and we must use the comment "!DELETE!".

https://docs.google.com/document/d/1UTm5JUZ8F9ptlcya9v88hzg-iPWxeuCCziAD_oycloo/edit?tab=t.0